test/zdtm/static/membarrier.c

```diff
@@ -0,0 +1,105 @@
+#include <linux/membarrier.h>
+#include <sys/syscall.h>
+#include "zdtmtst.h"
+
+const char *const test_doc = "Test membarrier() migration";
```

Comment:
Could you also add this test in [test/zdtm/static/Makefile](https://github.com/checkpoint-restore/criu/blob/criu-dev/test/zdtm/static/Makefile)?

Reply:
Done.